### PR TITLE
add timestamp to explicitly specified release for autorebuilds

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-7803

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by MartinBasti** [comment]
Please disable that pylint warning by inline comment, it seems false positive

File: tests/plugins/test_flatpak_create_oci.py [hunk 1]
```diff
@@ -543,7 +543,8 @@ def stream_to_generator(s):
         files = inspector.list_files()
         assert sorted(files) == config['expected_contents']
 
-        components = {c['name'] for c in workflow.image_components}
+        components = \
```
Comment (MartinBasti) [inline tests/plugins/test_flatpak_create_oci.py]: I'd say that in this case is better to violate pep8

File: atomic_reactor/schemas/container.json [hunk 2]
```diff
@@ -63,6 +63,10 @@
         "from_latest": {
           "description": "Whether to rebuild from the latest commit",
           "type": "boolean"
+        },
+        "add_timestamp_to_release": {
+          "description": "Whether to append timestamp to explicitly specified release",
```
[the remark this answers is not in the document]
Comment (rcerven) [reply atomic_reactor/schemas/container.json]: sure added

**Comment by twaugh** [comment]
Would be nice to document this in osbs-docs.

**Comment by rcerven** [comment]
> Would be nice to document this in osbs-docs.

yeah I'll add there also from_latest

**Comment by rcerven** [comment]
osbs-docs related PR https://github.com/containerbuildsystem/osbs-docs/pull/95

**Comment by twaugh** [comment]
I think we said we didn't need a feature flag for this, is that right?

**Comment by rcerven** [comment]
> I think we said we didn't need a feature flag for this, is that right?

nope, since it is basically flag by itself when enabled

**Comment by twaugh** [comment]
Coverage looks fine to me, not sure why coveralls reports it as a decrease.

**Comment by rcerven** [comment]
rn updated: add timestamp to explicitly specified release for autorebuilds

**Pull request merged**